joomla-projects/gsoc19_page-builder
# Improved Sidebar, colorpicker added

### Summary of Changes

Styling changes in sidebar to improve usability. 

Added color picker to sidebar to provide the option to chose Background Color, Font Color and Link Color. Link color isn't fully implemented yet. The option is only avaliable when opening the settings sidebar for a column. 
To render the frontend we extended the com_template plugins which handle the rendering of the pagebuilder settings to render now also inline css.

### Testing Instructions

1. Open the Pagebuilder in the apodis - Default template

2. Add a grid with any amount of columns 

3. Add content to at least on of the columns (alternatively you can also increase the height of the column on the site manually to see the changes to the background color)

4. Open the settings sidebar of a column 

5. Chose any color in the colorpicker for example for the background color

6. Save the template and set it as your default template

7. Open the frontend

### Expected result

The color changes to the specific columns are displayed.

### Actual result

Colorpicker doesn't exist in the sidebar.

### Documentation Changes Required

Not that I know.
